lib/workers/repository/extract/extract-fingerprint-config.ts

```diff
@@ -0,0 +1,79 @@
+import type { RenovateConfig } from '../../../config/types';
+import { get, getManagerList } from '../../../modules/manager';
+import type { WorkerExtractConfig } from '../../types';
+
+export interface FingerprintExtractConfig {
+  managerList: string[];
+  managers: WorkerExtractConfig[];
+}
+
+function getFilteredManagerConfig(
+  config: RenovateConfig,
+  manager: string
+): WorkerExtractConfig {
+  // the type here is not completely correct because there isn't any defined type only for managers
+  const managerConfig: WorkerExtractConfig = config[manager] as any;
+  const language = get(manager, 'language');
+  // the type here is not completely correct because there isn't any defined type only for languages
+  const languageConfig = (
+    language ? (config[language] ? config[language] : {}) : {}
+  ) as WorkerExtractConfig;
+  const filteredConfig = {} as WorkerExtractConfig;
+
+  filteredConfig.manager = manager;
+  filteredConfig.npmrc = config?.npmrc;
+  filteredConfig.npmrcMerge = config?.npmrcMerge;
```

Comment:
```suggestion
  filteredConfig.npmrc = config.npmrc;
  filteredConfig.npmrcMerge = config.npmrcMerge;
```
can't undefined or null here

Reply:
Why?